src/cases/case-feature.routes.ts

```diff
@@ -37,6 +44,35 @@ export const ROUTES: Routes = [
             title: 'HMCTS Share Cases | Case Share'
           }
         },
+        {
+          path: 'allocate-role',
+          canActivate: [HealthCheckGuard],
+          data: {
+            title: 'Allocate role'
+          },
+          children: [
+            {
+              path: 'exclusion',
+              component: AllocateRoleContainerComponent,
+            },
+            {
+              path: 'describe-exclusion',
+              component: AllocateRoleContainerComponent,
+            },
+            {
+              path: 'exclusion-self',
```

Comment:
I'm getting rid of this in my branch so would prefer if you did as well please

Reply:
cheers

Reply:
path: 'exclusion-self' should not be present

Reply:
Also you dnt need this path             {
              path: 'describe-exclusion',
              component: AllocateRoleContainerComponent,
            },